api/v6/report_server.thrift

```diff
@@ -68,7 +68,8 @@ enum SortType {
   CHECKER_NAME,
   SEVERITY,
   REVIEW_STATUS,
-  DETECTION_STATUS
+  DETECTION_STATUS,
```

Comment:
Does the comment about "comma after last item" hold here? That would reduce the number of line changes in enums/structs too, assuming it's not a syntax error.